lib/matplotlib/colors.py

```diff
@@ -64,9 +64,12 @@
 from collections import Sized
 import re
 import warnings
+from colorsys import rgb_to_hls, hls_to_rgb
```

Comment:
Importing these publically is a bad idea, because it makes `matplotlib.colors.rgb_to_hsv` and `matplotlib.colors.rgb_to_hls` look similar, when they have completely different argument conventions